jobs/scrape_archives.py

```diff
@@ -14,13 +14,20 @@
 
 ARCHIVE_URL = "https://archive.mozilla.org/pub/"
 PRODUCTS = ("fennec", "firefox", "thunderbird")
-FILE_EXTENSIONS = "zip|gz|bz|bz2|dmg|apk"
+FILE_EXTENSIONS = "zip|gz|bz2|dmg|apk"
 DEFAULT_SERVER = "https://kinto-ota.dev.mozaws.net/v1"
 DEFAULT_BUCKET = "build-hub"
 DEFAULT_COLLECTION = "archives"
 NB_THREADS = 3
 NB_RETRY_REQUEST = 3
 TIMEOUT_SECONDS = 5 * 60
+KNOWN_MIMETYPES = {
+    'apk': 'application/vnd.android.package-archive',
+    'bz2': 'application/x-bzip2',
```

Comment:
👍 

Even though cloudfront seems to give `application/x-tar` (`curl -I https://archive.mozilla.org/pub/firefox/nightly/2017/05/2017-05-02-10-02-49-mozilla-central/firefox-55.0a1.en-US.linux-i686.tar.bz2`)